src/Glob.cc

```diff
@@ -0,0 +1,64 @@
+#include "podio/utilities/Glob.h"
+#include <iostream>
+#include <stdexcept>
+#include <string>
+#include <vector>
+#if __has_include(<glob.h>)
+  #include <glob.h>
+#else
+  #include <system_error>
+#endif
```

Comment:
```suggestion
#if __has_include(<glob.h>)
  #include <glob.h>
  #define PODIO_HAS_GLOB_SUPPORT 1
#else
  #include <system_error>
  #define PODIO_HAS_GLOB_SUPPORT 0
#endif
```
Just to make it possible to have some checks downstream on this.

Reply:
added in header